README.md

```diff
@@ -136,6 +136,8 @@ more Emacs-y.
    [Rust installed](https://www.rust-lang.org/en-US/install.html). 
    The file `rust-toolchain` indicates the version that gets installed.
    This happens automatically, so don't override the toolchain manually.
+   IMPORTANT: When the toolchain was updated, you have to reinstall
+   rustfmt manually. PRs to fix this issue are very welcome.
```

Comment:
There is no change we can make. This is a failure in `rustup`.

Reply:
I thought this maybe could be bypassed somehow.

Reply:
I suspect the -preview items are not updated automatically. So as long as we use nightly and rustfmt/clippy stay preview this is way it is.